tools/setup/generate_landing_page_images.py

```diff
@@ -33,7 +37,6 @@ def generate_landing_page_images() -> None:
             size_1x = get_x_size(image.size, 1)
 
             ## Generate WEBP images.
-            image.save(f"{GENERATED_IMAGES_DIR}/{file_name}-3x.webp", quality=50)
```

Comment:
@terpimost it would be nice to note why we decided to remove the 3x images.

Reply:
just unnecessary big. 2x would look fine.